marimbabot_planning/src/planning.cpp

```diff
@@ -85,17 +85,17 @@ moveit::planning_interface::MoveGroupInterface::Plan Planning::plan_to_mallet_po
     tf2::Vector3 goal_position(
         goal_point.point.x,
         goal_point.point.y,
-        goal_point.point.z);
+        goal_point.point.z + 0.1);
```

Comment:
Offsetting the goals and changing the physical position of the marimba is a dirty hack to avoid collisions, which should not be done normally.